src/readFile.jl

```diff
@@ -0,0 +1,19 @@
+function readFile(fname)
+    s = open(fname) do file
+        read(file,String)
+    end
+end
+
+function getSection(s::String, section::String)
+    if section != "A"
+        exp = Regex("~(?=$section)(?s)(.*?)(?=~)")
+    else
+        exp = Regex("(?=~A)(?s).*")
+    end
+    try
+        data = match(exp,s)
+        return string(data.match)
+    catch
+        return ""
+    end
+end
```

Comment:
You don't need a separate file for this yet. Let's keep the code in `WellIO.jl` and then we can split it later if needed. There should be a simple `load` function that returns a `struct` with the fields agreed in the `1.2.md` spec.

Reply:
Okay!